packages/libretro/retroarch/system.d/tmp-system.mount

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=System directory
+Before=retroarch.target
+After=storage.mount
+After=systemd-tmpfiles-setup.service
+
+[Mount]
+What=none
+Where=/tmp/system
+Type=overlay
+Options=lowerdir=/usr/share/retroarch-system,upperdir=/storage/system,workdir=/storage/.tmp/system-workdir
```

Comment:
@natinusala This unit will fail to start in projects without PPSSPP, like RPi, as there will be no `/usr/share/retroarch-system` directory

Reply:
I miss clicked I didn't mean to resolve it

I have a fix for that income